pkg/systemd/sd_notify.go

```diff
@@ -6,9 +6,10 @@ import (
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
+// ErrSdNotifyNoSocket no socket was specified
+var ErrSdNotifyNoSocket = errors.New("No socket")
 
-// Send a message to the init daemon. It is common to ignore the error.
+// SdNotify Send a message to the init daemon. It is common to ignore the error.
```

Comment:
"SdNotify sends a message to the init daemon. It is common to ignore the return value."